src/routines.ts

```diff
@@ -108,14 +108,16 @@ export class SRPRoutines {
   }
 
   public async computeClientEvidence(
-    _I: string,
-    _s: bigint,
+    I: string,
+    s: bigint,
     A: bigint,
     B: bigint,
     S: bigint,
   ): Promise<bigint> {
     return arrayBufferToBigInt(
       await this.hash(
+        stringToArrayBuffer(I),
```

Comment:
WHy is this needed? I think RFC only says hash A and B values for evidence. Notice that we can use any known to both client and server values here without impacting anything.

Reply:
I don't really know, so I will change it back.